draft-yasskin-http-origin-signed-responses.md

```diff
@@ -941,35 +956,37 @@ Because different clients send different request header fields, and intermediate
 servers add response header fields, it can be impossible to have a signature for
 the exact request and response that the client sees. Therefore, when a client
 validates the `Signature` header field for an exchange represented as a normal
-HTTP request/response pair, it MUST pass only the subset of header fields
-defined by {{significant-headers}} to the validation procedure
-({{signature-validity}}).
+HTTP request/response pair, it MUST pass the serialized headers defined by
+{{serialized-headers}} to the validation procedure ({{signature-validity}})
+along with the `Signature` header field and the response's payload.
 
 If the client relies on signature validity for any aspect of its behavior, it
 MUST ignore any header fields that it didn't pass to the validation procedure.
```

Comment:
Since we removed `{{significant-headers}}` concept, we might want to be pedantic here:

If the client relies on signature validity for any aspect of its behavior, it MUST ignore any header fields whose name are not listed in the `Signed-Headers` header field.